data/assets/bastille-mas-model.scene

```diff
@@ -0,0 +1,126 @@
+--[[	{
+	    Key = "Shift+s",
+	    Command = 
+	        propertyHelper.invert('Scene.MAS_MHD_Fieldlines.Renderable.Enabled') ..
+	        propertyHelper.invert('Scene.MAS_MHD_density.Renderable.Enabled') ..
+	        propertyHelper.invert('Scene.SunGlare.Renderable.Enabled'),
+	    Documentation = "Resets the subscene for showing the Sun volume rendering + fieldlines",
+	    Name = "Reset MHD back",
+	    GuiPath = "/CCMC/MHD",
+	    Local = false
+	},--]]
+    {
+        Key = "F6",
+        Command = propertyHelper.invert('Scene.MAS_MHD_density.Renderable.Enabled'),
+        Documentation = "Toggle volume",
+        Local = false
+    },
+    {
+        Key = "F7",
+        Command = propertyHelper.invert('Scene.MAS_MHD_Fieldlines.Renderable.Enabled'),
+        Documentation = "Toggle fieldlines",
+        Local = false
+    },
+	{
+        Key = "F8",
+        Command = propertyHelper.invert('Scene.SunGlare.Renderable.Enabled'),
+        Documentation = "Toggle sun glare",
+        Local = false
+    },
+    {
+        Key = "F9",
+        Command = propertyHelper.invert('Scene.Sun.Renderable.Enabled'),
+        Documentation = "Toggle sun",
+        Local = false
+    },
+    {
+        Key = "i",
+        Command = [[
+            textureList = openspace.globebrowsing.getLayers('Sun', 'ColorLayers');
+
+            if (textureIndex == nil) then 
+                textureIndex = 1;
+            end; 
+
+            textureIndex = textureIndex + 1
+            if (textureIndex >= #textureList) then
+                textureIndex = 1;
+            end;
+
+            if (textureIndex == 1) then
+                openspace.setPropertyValue("Scene.Sun.Renderable.Layers.ColorLayers.*.Enabled", false);
+                openspace.setPropertyValueSingle("Scene.Sun.Renderable.Layers.ColorLayers.Texture.Enabled", true);
+            else
+                openspace.setPropertyValue("Scene.Sun.Renderable.Layers.ColorLayers.*.Enabled", false);
+
+                str = "Scene.Sun.Renderable.Layers.ColorLayers.magnetogram-" .. textureIndex .. ".Enabled";
+                openspace.setPropertyValueSingle(str, true);
+            end;
+
+        ]],
+        Documentation = "Display next sun texture in list of textures",
+        Local = false
+    }
+}
+
+local sunAsset = asset.require('scene/solarsystem/sun/sun');    --heliosphere/mas/bastille_day/
+
+asset.onInitialize(function ()
+    openspace.time.setTime("2000 JUL 14 08:33:37.100")
+
+    sceneHelper.bindKeys(Keybindings)
+
+    openspace.setPropertyValueSingle("Scene.Sun.Renderable.Layers.ColorLayers.Texture.Enabled", false)
+    local str = "Scene.Sun.Renderable.Layers.ColorLayers.magnetogram-" .. textureIndex .. ".Enabled";
+    openspace.setPropertyValueSingle(str, true)
+    
+    openspace.setPropertyValueSingle('Scene.SunGlare.Renderable.Enabled', false)
+	
+	openspace.markInterestingNodes({
+        "Earth", "Sun1", "Moon", "FL_ENLIL_earth", "FL_MAS", "Mas_MHD"
```

Comment:
is Sun1 a real thing? if so maybe we can fix the file.

Reply:
no it was more of an idea i tried and forgot to rename it back